yarn-project/entrypoints/src/dapp_entrypoint.ts

```diff
@@ -20,38 +22,52 @@ export class DefaultDappEntrypoint implements EntrypointInterface {
     private version: number = DEFAULT_VERSION,
   ) {}
 
-  async createTxExecutionRequest(exec: ExecutionRequestInit): Promise<TxExecutionRequest> {
-    const { calls, fee, authWitnesses = [], capsules = [] } = exec;
+  async createTxExecutionRequest(
+    exec: ExecutionPayload,
+    fee: FeeOptions,
+    _options: TxExecutionOptions,
```

Comment:
Throw when options are populated?

Reply:
This is going to be heavily refactored, entrypoints are a mess